schema/draft/schema.ts

```diff
@@ -2144,7 +2144,7 @@ export interface ElicitRequestFormParams extends TaskAugmentedRequestParams {
   /**
    * The elicitation mode.
    */
-  mode: "form";
+  mode?: "form" | undefined;
```

Comment:
Do we need both `?` and `undefined`?

Given this is a new field, should it be `mode?: "form"`?